packages/datetime/test/timePickerTests.tsx

```diff
@@ -209,6 +209,156 @@ describe("<TimePicker>", () => {
         assertTimeIs(timePicker.state.value, 0, 0, 0, 0);
     });
 
+    describe("Time range - minTime and maxTime props", () => {
+        it("if defaultValue is smaller than minTime, minTime becomes initial time", () => {
+            renderTimePicker({
+                defaultValue: new Date(1995, 6, 30, 12, 30),
+                minTime: new Date(1995, 6, 30, 15, 30),
+                precision: TimePickerPrecision.MILLISECOND,
+            });
+
+            assertTimeIs(timePicker.state.value, 15, 30, 0, 0);
+        });
+
+        it("if defaultValue is greater than maxTime, maxTime becomes initial time", () => {
+            renderTimePicker({
+                defaultValue: new Date(1995, 6, 30, 20, 30),
+                maxTime: new Date(1995, 6, 30, 18, 30),
+                minTime: new Date(1995, 6, 30, 15, 30),
+                precision: TimePickerPrecision.MILLISECOND,
+            });
+
+            assertTimeIs(timePicker.state.value, 18, 30, 0, 0);
+        });
+
+        it("by default, I can select any time", () => {
```

Comment:
Per @giladgray's earlier comment, first-person test descriptions aren't standard for our codebase, but we can either change these before merging this PR or after.